schema/meter_provider.json

```diff
@@ -186,6 +185,116 @@
             ],
             "title": "OtlpMetric"
         },
+        "AggregationTemporalityPreferenceByInstrumentKind": {
+            "type": ["object", "null"],
+            "additionalProperties": false,
+            "properties": {
+                "counter": {
+                    "$ref": "#/$defs/AggregationTemporality"
+                },
+                "histogram": {
+                    "$ref": "#/$defs/AggregationTemporality"
+                },
+                "observable_counter": {
+                    "$ref": "#/$defs/AggregationTemporality"
+                },
+                "observable_gauge": {
+                    "$ref": "#/$defs/AggregationTemporality"
+                },
+                "observable_up_down_counter": {
+                    "$ref": "#/$defs/AggregationTemporality"
+                },
+                "up_down_counter": {
+                    "$ref": "#/$defs/AggregationTemporality"
+                }
+            }
+        },
+        "AggregationTemporalityPreference": {
+            "type": ["string", "null"],
+            "enum": [
+                "delta",
+                "cumulative",
+                "low_memory"
+            ]
+        },
+        "DefaultAggregationByInstrumentKind": {
+            "type": ["object", "null"],
+            "additionalProperties": false,
+            "properties": {
+                "counter": {
+                    "$ref": "#/$defs/Aggregation"
+                },
+                "histogram": {
+                    "$ref": "#/$defs/Aggregation"
+                },
+                "observable_counter": {
+                    "$ref": "#/$defs/Aggregation"
+                },
+                "observable_gauge": {
+                    "$ref": "#/$defs/Aggregation"
+                },
+                "observable_up_down_counter": {
+                    "$ref": "#/$defs/Aggregation"
+                },
+                "up_down_counter": {
+                    "$ref": "#/$defs/Aggregation"
+                }
+            }
+        },
+        "Aggregation": {
```

Comment:
Lifting this aggregation type def up from the view stream definition. Its now reused both for view stream config and in exporter default aggregation config.